packaging/linux/snap/snapcraft.yaml

```diff
@@ -0,0 +1,24 @@
+name: keybase
+version: master
+summary: Crypto for everyone!
+description: |
+  Keybase service and command line client.
+
+grade: devel # must be 'stable' to release into candidate/stable channels
+confinement: strict
+
+apps:
+  keybase:
+    command: keybase
+    plugs: [network, network-bind]
+
+parts:
+  client-go:
+    source: ../../go
+    plugin: go
+    go-importpath: github.com/keybase/client/go
+    go-packages: [github.com/keybase/client/go/keybase]
+    go-buildtags: [production]
+    after: [go]
+  go:
+    source-tag: go1.7.5
```

Comment:
my approach on these cases has been to run sed from the CI system before calling the snapcraft command, like https://github.com/elopio/ipfs-snap/blob/master/scripts/snap-in-docker.sh#L17

We have some ideas to avoid that extra step, but there's always a strong pull to turn snapcraft into a full templating engine. I think that would remove a lot of its simplicity, but it's an option. We always need more voices and more ideas, so again, it would be great to have your input here:
https://forum.snapcraft.io/t/snapcraft-version-scriptlets/106

The discussion started with the version field, then moved to have replaceable variables everywhere, and now it's moving back to the version. I'm not sure we will be able to make everybody happy :)